### PR TITLE
Update to egui 0.21.0

### PR DESCRIPTION
**Pull request by emilk**
This updated to egui 0.21.0.

I deleted the `Cargo.lock` files, since this is what is recommended for libraries: https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html

Let me know if you want me to revert that!

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,10 @@ readme = "README.md"
 path = "src/lib.rs"
 
 [dependencies]
-egui = "0.20"
+egui = { version = "0.21", default-features = false }
```
Comment (emilk) [inline Cargo.toml:15]: Small change here: egui has the `default_fonts` feature by default. It bundles the default egui fonts (hence the name).

Previously, `egui-notify` would implicitly "force" the `default_fonts` feature onto users of `egui-notify`, bloating their binaries if they were not using `default_fonts` themselves.

**Comment by emilk** [comment]
@ItsEthra this is now ready for review! I would appreciate if this could be merged and a new `egui-notify` published as soon as possible. 🙏

**Review by ItsEthra — APPROVED** [review]
Looks good to me

**Comment by emilk** [comment]
Thanks for the swift response ❤️

**Pull request merged**